applications/SwimmingDEMApplication/custom_strategies/strategies/bdf2_turbulent_schemeDEMCoupled.h

```diff
@@ -326,6 +321,32 @@ class BDF2TurbulentSchemeDEMCoupled : public BDF2TurbulentScheme<TSparseSpace, T
 
     }
 
+    /// Store the iteration results as solution step variables and update acceleration after a Newton-Raphson iteration.
+    /**
+     * @param rModelPart fluid ModelPart
+     * @param rDofSet DofSet containing the Newton-Raphson system degrees of freedom.
+     * @param A Newton-Raphson system matrix (unused)
+     * @param Dx Newton-Raphson iteration solution
+     * @param b Newton-Raphson right hand side (unused)
+     */
+    void Update(
+        ModelPart& rModelPart,
+        DofsArrayType& rDofSet,
+        TSystemMatrixType& A,
+        TSystemVectorType& Dx,
+        TSystemVectorType& b) override
+    {
+        KRATOS_TRY
+
+        double alpha = rModelPart.GetProcessInfo()[RELAXATION_ALPHA];
```

Comment:
which is the usual value for this? (pure curiosity :) )

Reply:
maybe you should check if this value has been set (maybe you're not setting it in all cases)

Reply:
This is set by default to 1.0 (no relaxation) and it is always set in the default project parameters. Anyways, as I talked with @rubenzorrilla, this PR will be split in several parts. At least three.